srv/components/misc_pkgs/kafka/files/zookeeper.properties

```diff
@@ -34,7 +34,7 @@ initLimit=10
 syncLimit=5
 dataLogDir=/var/log/zookeeper
 {%- for node in server_nodes %}
-server.{{ loop.index }}={{ pillar['cluster'][node]['hostname'] }}:2888:3888
+server.{{ loop.index }}={{ pillar['cluster'][node]['network']['data']['private_fqdn'] }}:2888:3888
```

Comment:
Can we place ports info is defined constant file (as config)

Reply:
We can, but why do we want to do that?
These are standard Kafka follower and server ports.

Reply:
Let me add that.

Reply:
Addressed this with moving the ports to confstore.